### PR TITLE
III-4250 Document update name endpoint for places and events

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Documented endpoint to update the name in the given language of a place
- Documented endpoint to update the name in the given language of an event

---

Ticket: https://jira.uitdatabank.be/browse/III-4250

### CODE REVIEW

File: projects/uitdatabank/reference/entry.json [hunk 1]
```diff
@@ -1853,6 +1853,59 @@
...
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/EventNotFound"
+          }
+        },
+        "operationId": "event-name-put",
+        "description": "Update the name of an event based on the given `eventId` and `language` inside the URL.",
```
Comment (bertramakers) [inline projects/uitdatabank/reference/entry.json:1888]:
```suggestion
        "description": "Updates the localized name of an event based on the given `eventId` and `language` inside the URL.",
```

File: projects/uitdatabank/reference/entry.json [hunk 2]
```diff
@@ -3709,6 +3762,59 @@
...
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/PlaceNotFound"
+          }
+        },
+        "operationId": "place-name-put",
+        "description": "Update the name of a place based on the given `placeId` and `language` inside the URL.",
```
Comment (bertramakers) [inline projects/uitdatabank/reference/entry.json:3797]:
```suggestion
        "description": "Updates the localized name of a place based on the given `placeId` and `language` inside the URL.",
```

File: projects/uitdatabank/reference/entry.json (hunk 1 above)
Comment (bertramakers) [reply projects/uitdatabank/reference/entry.json:1888]: We usually follow the style "<this endpoint> updates / creates / ..."

File: projects/uitdatabank/models/event-name-put.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "type": "object",
+  "title": "event.name.put",
+  "additionalProperties": false,
+  "minProperties": 1,
+  "maxProperties": 1,
```
Comment (bertramakers) [inline projects/uitdatabank/models/event-name-put.json:6]: Why the `minProperties` and `maxProperties`? That seems only relevant if you use `additionalProperties: {...}` (so with a schema, not `true`/`false`, to enable a variable list of properties names)

**Comment by bertramakers** [comment]
@LucWollants As per the README in this repository please remember to prefix your branch with uitdatabank/ for future changes, to avoid confusion what branch is relevant for what project. Thanks! :)

File: projects/uitdatabank/models/event-name-put.json (hunk 3 above)
Comment (LucWollants) [reply projects/uitdatabank/models/event-name-put.json]: Indeed, good point!

**Comment by LucWollants** [comment]
> @LucWollants As per the README in this repository please remember to prefix your branches with `uitdatabank/` for future changes, to avoid confusion what branch is relevant for what project. Thanks! :)

Oh yes, that is indeed a much better approach now.

**Review by bertramakers — APPROVED** [review]

**Comment by bertramakers** [comment]
@LucWollants before merging this, make sure to merge the latest changes from `main` into this branch and check that the linter is still green. (Because of the new Spectral rules)

**Pull request merged**